dev-docs/bidders/preciso.md

```diff
@@ -1,63 +1,129 @@
 ---
 layout: bidder
 title: Preciso
-description: Prebid Preciso Bidder Adapter
+description: Prebid Preciso Bid Adapter
+gdpr_supported: true
 gvl_id: 874
-media_types: display, video, native 
+media_types: display
 gdpr_supported: true
 usp_supported: true
 pbjs: true
-pbs: true
+pbs: false
 biddercode: preciso
 prebid_member: true
 floors_supported: true
 safeframes_ok: true
 schain_supported: true
-userIds: id5Id, identityLink, pubProvidedId
-pbs_app_supported: true
+userIds: sharedId
+deals_supported: false
 coppa_supported: true
-multiformat_supported: will-bid-on-any
-ortb_blocking_supported: partial
+multiformat_supported: false
+ortb_blocking_supported: true
 sidebarType: 1
 ---
 
+### Modules
 
-### Bid Params
+SharedID: We need you to include SharedID module,which is used to get prebid user commonid.It can better differentiating users to bid on ads.
```

Comment:
```suggestion
We need you to include the [SharedID module](/dev-docs/modules/userid-submodules/sharedid.html) in order to bid effectively on your inventory.
```

Please also remove the sample configuration for sharedId and only leave the custom filterSettings you require

Reply:
@muuki88  Thanks for updating me. I have done changes.